.travis.yml

```diff
@@ -0,0 +1,17 @@
+dist: xenial
+language: go
+
+git:
+  depth: 1
+
+services:
+  - docker
+before_script:
+  - CHANGED_FILES_PR=$(git diff --name-only HEAD $(git merge-base HEAD $TRAVIS_BRANCH))
+script:
+  - make ci e2e status=keep
+after_success:
+  - if [[ "${CHANGED_FILES_PR[@]}" =~ "scripts/kind-e2e/e2e.sh" ]]; then
+      echo "scripts/kind-e2e/e2e.sh was modified, testing recurring run on already deployed infrastructure.";
```

Comment:
what is this supposed to mean?

Reply:
Testing on recurring runs if we use make e2e status=keep